pkgs/applications/version-management/p4/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "p4";
+  version = "2020.1.1991450";
+
+  src = fetchurl {
+    url = "https://cdist2.perforce.com/perforce/r20.1/bin.linux26x86_64/helix-core-server.tgz";
+    sha256 = "0nhcxhwx3scx6vf7i2bc8j0b1l57lmq9bfy1cfbfbqasd3an721k";
+  };
+
+  # Work around the "unpacker appears to have produced no directories"
+  # case that happens when the archive doesn't have a subdirectory.
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp p4 p4broker p4d p4p $out/bin
+
+    for f in $out/bin/*; do
+      patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" $f
+    done
+  '';
+
+  meta = {
+    description = "Perforce Command-Line Client";
+    homepage = "https://www.perforce.com";
+    license = lib.licenses.unfreeRedistributable;
```

Maybe you can prove me wrong, but it looks to me like this is `unfree`:
```suggestion
    license = lib.licenses.unfree;
```

---

This was copied from `p4v`.  Unfortunately all I can find are licenses for dependencies (e.g. boost), which are all free.  I'll change it to `unfree`.

Perhaps we should consider changing p4v as well?

---

I also only found licenses for the dependencies, but downloading from the official site seems to require agreeing with EULA[1]. I would think the change to `p4v.meta.license` is warranted as well.

[1] https://www.perforce.com/terms-use#eula